liblabjackusb/labjackusb.c

```diff
@@ -627,39 +637,7 @@ int LJUSB_OpenAllDevices(HANDLE* devHandles, UINT* productIds, UINT maxDevices)
         }
 
         if (LJ_VENDOR_ID == desc.idVendor) {
-            // Found a LabJack device
-            r = libusb_open(dev, &devh);
-            if (r < 0) {
-                LJUSB_libusbError(r);
-                continue;
-            }
-
-            // Test if the kernel driver has the U12.
-            if (desc.idProduct == U12_PRODUCT_ID && libusb_kernel_driver_active(devh, 0) ) {
-#if LJ_DEBUG
-                fprintf(stderr, "Kernel Driver was active, detaching...\n");
-#endif
-
-                // Detach the U12 from kernel driver.
-                r = libusb_detach_kernel_driver(devh, 0);
-
-                // Check the return value
-                if ( r != 0 ) {
-                    fprintf(stderr, "failed to detach from kernel driver. Error Number: %i", r);
-                    libusb_close(devh);
-                    continue;
-                }
-            }
-
-            r = libusb_claim_interface(devh, 0);
-            if (r < 0) {
-                //LJUSB_libusbError(r);
```

Comment:
@sjarman28 also note a subtle behaviour change here... this commented out `//LJUSB_libusbError(r);` is essentially back.  Hopefully that's ok...

Reply:
thx. fixed.

Reply:
I think the behavior change is okay, it should just cause more debug output when debug is enabled.